plugin/trino-delta-lake/src/test/java/io/trino/plugin/deltalake/BaseDeltaLakeMinioConnectorTest.java

```diff
@@ -270,4 +273,37 @@ public void testCreateSchema()
         // verify DROP SCHEMA for non-existing schema
         assertQueryFails("DROP SCHEMA " + schemaName, format("line 1:1: Schema '.*\\.%s' does not exist", schemaName));
     }
+
+    @Test
+    @Override
+    public void testRenameTableUnqualifiedName()
+    {
+        // override needed to provide valid location for schema (it needs to be s3:// based). By default schema is assigned hdfs:// location
+        // TODO: better fix would be to configure hadoop container we use in test to use `s3://` filesystem by default
+        skipTestUnless(hasBehavior(SUPPORTS_CREATE_SCHEMA) && hasBehavior(SUPPORTS_CREATE_TABLE));
```

Comment:
add empty line after `// TODO: better fix would`, since the code line and comment are unrelated

Reply:
BTW `skipTestUnless` is not needed in a specific test class (here), because it knows what it does and doesn't support

Reply:
Not applying anymore because the duplicated test method has been removed from the class.